.github/workflows/push-mimir-build-image.yml

```diff
@@ -70,14 +70,13 @@ jobs:
         id: notification
         run: | 
           if [ ${{ steps.check_build.outputs.build }} == 'true' ]; then
-           gh pr comment $PR_URL --body "**Building new version of mimir-build-image**. After image is built and pushed to Docker Hub, \
+           gh pr comment ${{ github.event.pull_request.number }} --body "**Building new version of mimir-build-image**. After image is built and pushed to Docker Hub, \
```

Comment:
nit: This command already uses `$IMAGE:$TAG`, it would be nice to be consistent and use $PR_NUMBER, instead of introducing another way of accessing the variables.

(Nice catch about `$PR_URL` though, it really seems that it wasn't defined)

Reply:
Done.